bulbs/super_features/filters.py

```diff
@@ -0,0 +1,22 @@
+import django_filters
+from rest_framework import filters
+
+from bulbs.super_features.utils import get_superfeature_model
+
+SUPERFEATURE_MODEL = get_superfeature_model()
+
+
+def filter_status(queryset, value):
```

Comment:
note: this isn't my favorite thing ever, but status is currently a property (that is different site side vs. bulbs side), so this seemed like an ok way to do this, unless we want to move status to being a model field for `Content`